lib/stage-autocompleter.js

```diff
@@ -121,9 +121,42 @@ class StageAutoCompleter {
    *
    * @returns {Function} The completion function.
    */
-  getCompletions(editor, session, position, prefix, done) {
+  async getCompletions(editor, session, position, prefix, done) {
```

Comment:
We cannot use async/await as this will not be compatible with Cloud. They need to support IE.